src/AltinnCore/Common/Services/Implementation/GiteaAPIWrapper.cs

```diff
@@ -396,7 +387,7 @@ public async Task<string> GetUserNameFromUI()
                     string htmlContent = await response.Content.ReadAsStringAsync();
                     string token = GetStringFromHtmlContent(htmlContent, "<div class=\"ui info message\">\n\t\t<p>", "</p>");
                     List<string> keys = FindAllAppKeysId(htmlContent, keyName);
-                    _logger.LogInformation("The number of app keys matching keyname " + keyName + " is " + keys.Count());
+                    _logger.LogInformation("The number of app keys matching keyname " + keyName + " is " + keys.Count);
```

Comment:
may be update this also with string interpolation

Reply:
fixed